src/app/components/client/toolbar/UserMenu.tsx

```diff
@@ -105,7 +105,7 @@ export const UserMenu = (props: UserMenuProps) => {
       >
         <a
           className={styles.menuItemCta}
-          href={process.env.NEXT_PUBLIC_EXTERNAL_SUPPORT_URL}
+          href={process.env.NEXT_PUBLIC_MONITOR_SUPPORT_URL}
```

Comment:
This ensures it uses the value from https://github.com/mozilla/blurts-server/blob/d1a7ce612f8976d8bfef56a29214ada527914822/.env-dist#L150.